client/app/constants/token.js

```diff
@@ -0,0 +1,3 @@
+export const TOKEN = {
+  ADD_TOKEN: 'ADD_TOKEN'
```

Comment:
nitpick: I think that TOKEN.ADD would suffice, not sure if we need to be that verbose.